pages/admin/admin-management.tsx

```diff
@@ -63,29 +67,35 @@ export default function AdminManagement() {
     {
       Header: 'Name',
       accessor: 'name',
-      sortDescFirst: true,
+      Cell: ({ value }) => {
```

Comment:
```suggestion
      Cell: ({ value }: { firstName: string; lastName: string; }) => {
```
Cause of failing build

Reply:
the error still occurs for some reason

Reply:
I went with another solution I got from `pages/admin/index.ts`

Reply:
Sorry what is the solution?

Reply:
oh I realized you can't see it here but you can see it in the Files Changed section. Basically, it was importing the Column type from `react-table` and setting the `COLUMNS` array to be of type `Column<any>[]` and then the error goes away